mkdocs.yaml

```diff
@@ -11,12 +11,11 @@ nav:
     - tutorial/converting-data-to-nexus.md
     - tutorial/nexus-to-nomad.md
   - How-tos:
-    - how-tos/writing-an-appdef.md
```

Comment:
these should not be removed (even if they are not yet fully fleshed out), I readded them

Reply:
We can leave them in the tree on the left-hand side and say in index.md that they are under development

Reply:
Okay, I put intially somewhere a note that this is deveopled, but not yet ready. If it should be this way, Im fine with that. Just doesn't look profesionally if someone visits this site and sees that 3 guides are construction sites (not sure for how this state is).
anyway, then lets keep it.